regimes/pl/README.md

```diff
@@ -53,12 +53,12 @@ The following GOBL maps to the `1` (gotówka) value of the `TFormaPlatnosci` fie
 
 All Polish invoices must be identified with a specific type code defined by the FA_VAT format. The following table helps identify how GOBL will map the expected Polish code with a combination of the Invoice Type and tax tags.
 
-| Code    | Type         | Tax Tags     | Description              |
-| ------- | ------------ | ------------ | ------------------------ |
-| VAT     | `standard`   |              | Regular invoice          |
-| UPR     | `standard`   | `simplified` | Simplified (no customer) |
-| ZAL     | `standard`   | `partial`    | Advance invioce          |
-| ROZ     | `standard`   | `settlement` | Settlement invoice       |
-| KOR     | `corrective` |              | Corrective (regular)     |
-| KOR_ZAL | `corrective` | `partial`    | Corrective (advance)     |
-| KOR_ROZ | `corrective` | `settlement` | Corrective (settlement)  |
+| Code    | Type          | Tax Tags     | Description               |
+| ------- | ------------- | ------------ | ------------------------- |
+| VAT     | `standard`    |              | Regular invoice           |
+| UPR     | `standard`    | `simplified` | Simplified (no customer)  |
+| ZAL     | `standard`    | `partial`    | Advance invioce           |
+| ROZ     | `standard`    | `settlement` | Settlement invoice        |
+| KOR     | `credit-note` |              | Credit note (regular)     |
+| KOR_ZAL | `credit-note` | `partial`    | Credit note (advance)     |
```

Comment:
It might be worth clarifying here if possible. Is this a "partial" credit note, or a credit note of a partial invoice?

Reply:
Good question :D. This is a credit note of a partial invoice